### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by ridjis**
https://trello.com/c/MiwK8jXM

image

image

image

### CODE REVIEW

File: components/sign_out.tsx
Comment (ridjis) [inline components/sign_out.tsx:1]: Moved logic sign out button used to navbar component.

File: components/cv_list.tsx
Comment (ridjis) [inline components/cv_list.tsx:1]: In this file, I've only removed button that was last element in this component (and hence removed fragment wrapper - those are only two changes.

File: components/sign_in_form.tsx
Comment (ridjis) [inline components/sign_in_form.tsx:1]: Changed color and added wrapper around form control elements and error messages.

File: components/ui/button.tsx [hunk 1]
```diff
@@ -51,38 +52,56 @@ export const buttonClasses = cva(
...
-      onClick={onClick}
-      type={type}
-    >
-      {prefix}
-      {children}
-      {suffix}
-    </button>
-  );
-};
+const Button = React.forwardRef(
```
Comment (ridjis) [inline components/ui/button.tsx:68]: Wrapped component in forwardRef so headlessui's `Menu.Item` component (and later others if necessary) can pass ref to DOM `button` element.

File: tailwind.config.js [hunk 2]
```diff
@@ -3,6 +3,9 @@ const colors = require("tailwindcss/colors");
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   content: ["./app/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
+  // this should disable auto dark mode application if that's system preference
```
Comment (mvukovicbdit) [inline tailwind.config.js:6]: Confirming it works, I'm on dark mode, but the app is on the light.

**Pull request merged**